src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -293,11 +293,20 @@
                         const area_id = feature.properties && feature.properties["area_id"];
                         const area_name = feature.properties && feature.properties["area_name"];
 
+                        console.log(featureIndicators[area_id])
                         const values = featureIndicators[area_id];
                         const value = values && values!.value;
-
+                        const lower_value = values && values!.lower_value;
+                        const upper_value = values && values!.upper_value;
                         const stringVal = (value || value === 0) ? value.toString() : "";
-                        if (stringVal) {
+                        if (stringVal && lower_value > 0) {
```

Comment:
Couldn't a lower value in a range legitimately be 0? Would it be safer here to switch on whatever falsy value you actually get for values without ranges (the input data)?

Reply:
Modified to accept 0 as a starting value

Reply:
I have also added the ranges to bubble map with help from Jeff